pytket/extensions/cutensornet/general.py

```diff
@@ -81,14 +81,14 @@ def __exit__(self, exc_type: Any, exc_value: Any, exc_tb: Any) -> None:
     def print_device_properties(self, logger: Logger) -> None:
         """Prints local GPU properties."""
         device_props = cp.cuda.runtime.getDeviceProperties(self.dev.id)
-        logger.debug("===== device info ======")
-        logger.debug("GPU-name:", device_props["name"].decode())
-        logger.debug("GPU-clock:", device_props["clockRate"])
-        logger.debug("GPU-memoryClock:", device_props["memoryClockRate"])
-        logger.debug("GPU-nSM:", device_props["multiProcessorCount"])
-        logger.debug("GPU-major:", device_props["major"])
-        logger.debug("GPU-minor:", device_props["minor"])
-        logger.debug("========================")
+        logger.info("===== device info ======")
+        logger.info("GPU-name: " + device_props["name"].decode())
```

Comment:
Purely nitpicking: why not opt for fstrings?

E.g.
```
logger.info(f"GPU-name: {device_props['name'].decode()}")
```

Reply:
I went for that in the first commit, and mypy did not know how to parse that for some reason, so I went for this other approach 🤷